src/orca-loader/PaymentSessionMethods.res

```diff
@@ -1,3 +1,17 @@
+let sortFunctions = (a, b) => {
+  open Utils
+  let temp1 = Date.fromString(a->getDictFromJson->getString("last_used_at", ""))
+  let temp2 = Date.fromString(b->getDictFromJson->getString("last_used_at", ""))
+
+  if temp1 === temp2 {
+    0.
+  } else if temp1 > temp2 {
+    -1.
+  } else {
+    1.
+  }
+}
+
```

Comment:
Will move this function out when this PR will be merged - https://github.com/juspay/hyperswitch-web/pull/427